srslie/fitlit
# Charts

**Changes:**
- Add hard-coded graphs for all activities
- Add graph.js file to work on refactor graph display
- Reorganize some of scripts

**Issues:**
- Water graph is not displaying for some reason, data should be right and the dates are displaying so it's weird
- Very wet code
-  Maybe we can get rid of some global query selectors if we only use them once?
- I want to start refactoring things into some bigger functions for one-line displays
- Graph colors and stuff are ugly, need to style

- [ ] DRY and SRP


**How to Review Changes or Manually Test Code:**
- open index.html
- check in scripts.js for code changes

**Notes:**
- honestly not sure why water isn't working, I'm still chugging away at doing this but this at least gives you more to work with for styling, media queries, accessibility stuff
